bedrock/products/templates/products/vpn/includes/subnav.html

```diff
@@ -14,27 +14,15 @@ <h2 class="c-sub-navigation-title is-summary">
           </a>
         </h2>
         <ul class="c-sub-navigation-list is-details is-closed">
+
           <li class="c-sub-navigation-item">
-            <a href="{{ url('products.vpn.more.what-is-a-vpn') }}" data-link-type="nav" data-link-position="subnav" data-link-name="What’s a VPN?">
-              {{ ftl('vpn-subnav-whats-a-vpn') }}
-            </a>
-          </li>
-          {% if show_relay_promo_block %}
-            <li class="c-sub-navigation-item">
-              <a href="#bundle" data-link-type="nav" data-link-position="subnav" data-link-name="Why bundle?">
-                {{ ftl('vpn-shared-why-bundle') }}
-              </a>
-            </li>
-          {% endif %}
-          <li class="c-sub-navigation-item">
-            {% set faq_link = '#faq' if request.path == '/products/vpn/' else url('products.vpn.landing') + '#faq' %}
-            <a href="{{ faq_link }}" data-link-type="nav" data-link-position="subnav" data-link-name="FAQs">
-              {{ ftl('vpn-subnav-faqs') }}
+            <a href="{{ url('products.vpn.resource-center.landing') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Learn about VPNs?">
```

Comment:
Nit: remove leftover `?`?
```suggestion
            <a href="{{ url('products.vpn.resource-center.landing') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Learn about VPNs">
```